laboratorios/internal/views/laboratory.py

```diff
@@ -192,8 +193,7 @@ def create(request,
                     messages.error(
                         request, 'Este nombre de laboratorio ya existe, pruebe otro')
                     return redirect('internal:laboratory.create')
```

Comment:
Este `return` debería estar al nivel del `for`, si por alguna razón sale un error que no corresponde con lo que esperan, nunca devuelve nada y el servidor se cae (porque un `view` deber devolver un `HttpResponse`